packages/patternfly-4/react-core/src/experimental/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -15,7 +15,7 @@ export const DrawerPanelContent: React.SFC<DrawerPanelContentProps> = ({
   ...props
 }: DrawerPanelContentProps) => (
       <aside className={css(styles.drawerPanel, className)} {...props}>
-        <div className={css(styles.drawerPanelBody)}>
+        <div className={css(styles.drawerPanelBody, styles.modifiers.noPadding)}>
```

Comment:
looks like this just adds the modifier? It should be a prop you can define for the panel content that applies this class. Without the modifier defined (default), the panel should have padding. With the modifier applied, the panel should have no padding around it (as it does right now).

FWIW, we'll be adding the same type of prop for the drawer content element, too, but I need to make that change in core first.

Reply:
okay will make this update